R/SubsetDefinitions.R

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+.defaultNameTemplate = "@baseCohortName - @subsetDefinitionName @operatorNames"
```

Comment:
Curious as to why this is declared outside of the scope of the R6Class definition?

Reply:
Noting that this is an internal package constant and makes the class function that uses its value simpler.